Test fails: FAIL: test_simple_dict_get_second_element (test_to_dict.TestToDict)

```
FAIL: test_simple_dict_get_second_element (test_to_dict.TestToDict)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/runner/work/scapy_helper/scapy_helper/test/test_to_dict.py", line 28, in test_simple_dict_get_second_element
    self.assertEqual(to_dict_result, packet_result)
AssertionError: {'IP': {'version': 4, 'ihl': None, 'tos': 0, 'len'[132 chars] []}} != {'IP': {'frag': 0, 'src': '0.0.0.0', 'proto': 6, '[130 chars]: 1}}
  {'IP': {'chksum': None,
          'dst': '127.0.0.1',
          'flags': None,
          'frag': 0,
          'id': 1,
          'ihl': None,
          'len': None,
          'options': [],
          'proto': 6,
-         'src': '127.0.0.1',
?                 ^^^     ^

+         'src': '0.0.0.0',
?                 ^     ^

          'tos': 0,
          'ttl': 64,
          'version': 4}}
```

--- a/test/test_to_dict.py
+++ b/test/test_to_dict.py
@@ -8,7 +8,7 @@ from scapy_helper.helpers.to_dict import to_dict, to_list
 
 class TestToDict(TestCase):
     def test_simple_dict(self):
-        packet = Ether(dst="ff:ff:ff:ff:ff:ff", src="00:00:00:00:00:00") / IP() / TCP()
+        packet = Ether(dst="ff:ff:ff:ff:ff:ff", src="00:00:00:00:00:00") / IP(src="0.0.0.0", dst="127.0.0.1") / TCP()
         packet_result = {'Ethernet': {'src': '00:00:00:00:00:00', 'dst': 'ff:ff:ff:ff:ff:ff', 'type': 2048}}
 
         to_dict_result = to_dict(packet)
@@ -17,7 +17,7 @@ class TestToDict(TestCase):
         self.assertEqual(to_dict_result, packet_result)
 
     def test_simple_dict_get_second_element(self):
-        packet = Ether(dst="ff:ff:ff:ff:ff:ff", src="00:00:00:00:00:00") / IP() / TCP()
+        packet = Ether(dst="ff:ff:ff:ff:ff:ff", src="00:00:00:00:00:00") / IP(src="0.0.0.0", dst="127.0.0.1") / TCP()
         packet_result = {'IP': {'frag': 0, 'src': '0.0.0.0', 'proto': 6, 'tos': 0, 'dst': '127.0.0.1',
                                 'chksum': None, 'len': None, 'options': [], 'version': 4, 'flags': None,
                                 'ihl': None, 'ttl': 64, 'id': 1}}
@@ -30,7 +30,7 @@ class TestToDict(TestCase):
 
 class TestToList(TestCase):
     def test_simple_list(self):
-        packet = Ether(dst="ff:ff:ff:ff:ff:ff", src="00:00:00:00:00:00") / IP() / TCP()
+        packet = Ether(dst="ff:ff:ff:ff:ff:ff", src="00:00:00:00:00:00") / IP(src="0.0.0.0", dst="127.0.0.1") / TCP()
         packet_result = [{'Ethernet': {'src': '00:00:00:00:00:00', 'dst': 'ff:ff:ff:ff:ff:ff', 'type': 2048}}, {
             'IP': {'frag': 0, 'src': '0.0.0.0', 'proto': 6, 'tos': 0, 'dst': '127.0.0.1', 'chksum': None, 'len': None,
                    'options': [], 'version': 4, 'flags': None, 'ihl': None, 'ttl': 64, 'id': 1}}, {

